scheduler/src/cook/rest/api.clj

```diff
@@ -2018,22 +2047,24 @@
      [false {::error \"...\"}]
 
   where \"...\" is a detailed error string describing the quota bounds exceeded."
-  [conn {:keys [::jobs ::pool] :as ctx}]
+  [conn {:keys [::jobs ::job-pool-name-maps] :as ctx}]
```

Comment:
Isn't ::jobs dead here?  (And shouldn't it be dead?)

Reply:
Good call, removed